### DISCUSSION TITLE
Is it possible to configure a SSL/TLS connection to mongoose?

### DISCUSSION THREAD

**Discussion by diego-rapoport** [TypedProject/ts-express-decorators#1941] [Q&A]
Well, I have a replicaset that it's configured to connect via TLS. As far as I saw in the `ConnectOptions` I can only set user and password to connect. Is there any other place where I can configure for tls connection or it's not yet implemented?
If not yet implemented, is there a will for this kind of implementation?

**Comment by Romakita** [comment]
The typing is maybe wrong bu you can ignore it and give the options for a classic mongoose app.

**Reply by diego-rapoport** [reply]
Ok! Will give it a try. Thanks!

**Reply by Romakita** [reply]
Just read the mongoose documentation to create a tls connection ;)

**Reply by diego-rapoport** [reply]
Thank you very much, I already did it.
